README.md

```diff
@@ -22,14 +22,15 @@ Visit the [BOPTEST Home Page](https://ibpsa.github.io/project1-boptest/) for mor
 ## Quick-Start to Deploy a Test Case
 1) Download this repository.
 2) Install [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/).
-3) Build and deploy a test case using the following commands executed in the root directory of this repository and where <testcase_dir_name> is the name of the test case subdirectory located in [/testcases](https://github.com/ibpsa/project1-boptest/tree/master/testcases):
+3) To build and deploy a test case, use the following commands within the root directory of the extracted software:
 
-  * Linux or macOS: ``$ TESTCASE=<testcase_dir_name> docker-compose up``
-  * Windows PowerShell: ``> ($env:TESTCASE="<testcase_directory>") -and (docker-compose up)``
+  * Linux or macOS: ``$ TESTCASE=<testcase_name> docker-compose up``
+  * Windows PowerShell: ``> ($env:TESTCASE="<testcase_name>") -and (docker-compose up)``
   * A couple notes:
-    * The first time this command is run, the image ``boptest_base`` will be built.  This takes about a minute.  Subsequent usage will use the already-built image and deploy much faster.
+    * Replace ``<testcase_name>`` with the name of the test case you wish to deploy.
```

Comment:
Suggest adding where a user can find the test case names.  So maybe something like, "Replace ``<testcase_name>`` with the name of the test case you wish to deploy.  Test case names can be found at https://github.com/ibpsa/project1-boptest/tree/master/testcases or on the web page at https://ibpsa.github.io/project1-boptest/testcases/index.html."

Reply:
And maybe replace the raw link text with words like, "...can be found in the [/testcases](link) directory of the repo or in the [Test Cases](link) tab of the web page."

Reply:
yep, seems reasonable to me!

Reply:
Ok if I write directly to your fork branch to update this PR accordingly?

Reply:
please do